content/en/docs/pipelines/sdk/python-function-components.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<blockquote> the container &nbsp;image must use Python 3.5 or later. </blockquote><p>Just FYI I think we're deprecating Py 3.5 support. Shall we directly put Python 3.6 here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/website/pull/2290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>